src/Common/src/Internal/Runtime/ModuleHeaders.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.Runtime
+{
+    //
+    // ModuleHeaderSection IDs are used by the runtime to look up specific global data sections
+    // from each module linked into the final binary. New sections should be added at the bottom
+    // of the enum and deprecated sections should not be removed to preserve ID stability.
+    //
+    enum ModuleHeaderSection
+    {
+        StringEETypePtr             = 0x00,
+        StringTableStart            = 0x01,
```

Comment:
We should have only one row per section - that has both start and end.

Something like:
```
struct ModuleInfoRow {
    int32 SectionID;
    int32 Flags; // Eventually, these flags can say whether start/end are actual pointers, RVAs, GP-relative, ...
    void * Start;
    void * End;
};
```